test/BarnPrizePool.test.js

```diff
@@ -0,0 +1,140 @@
+const { deployContract } = require('ethereum-waffle')
+const { deployMockContract } = require('./helpers/deployMockContract')
+const BarnPrizePoolHarness = require('../build/BarnPrizePoolHarness.json')
+const TokenListenerInterface = require('../build/TokenListenerInterface.json')
+const ControlledToken = require('../build/ControlledToken.json')
+const BarnFacetMock = require('../build/BarnFacetMock.json')
+const BarnBridgeToken = require('../build/BarnBridgeToken.json')
+
+const { ethers } = require('ethers')
+const { expect } = require('chai')
+const buidler = require('@nomiclabs/buidler')
+
+const toWei = ethers.utils.parseEther
+
+const debug = require('debug')('ptv3:yVaultPrizePool.test')
+
+let overrides = { gasLimit: 20000000 }
+
+describe('BarnPrizePool', function () {
```

Comment:
I would like to see the following tests:
1. A test that calls the depositTo method of the prizePool and checks the deposited balances
2. A test that calls withdraw after the campaign is over and gets you back your BOND
3. A test that showcases the award is sent to the winner
4. A test that showcases that the `balanceOf` method of the Prize pool returns the interest accrued too